hw2/03-count.js

```diff
@@ -1,3 +1,127 @@
-// Add your code here
+/* 
+  Program to highlight matching words for the given paragraph below.
+  This paragraph would be part of a server api call but is statically
+  typed here for front end testing of the functions
+*/
 
-input.addEventListener('keydown', handleKeyDown);
+const originalParagraph = `It was the best of times, it was the worst of times, it was the age of
+wisdom, it was the age of foolishness, it was the epoch of belief, it
+was the epoch of incredulity, it was the season of Light, it was the
+season of Darkness, it was the spring of hope, it was the winter of
+despair, we had everything before us, we had nothing before us, we were
+all going direct to Heaven, we were all going direct the other way—in
+short, the period was so far like the present period, that some of its
+noisiest authorities insisted on its being received, for good or for
+evil, in the superlative degree of comparison only.`;
+
+// Standard message error
+const m_error = {
+  message: "Invalid input, please enter a word to highlight",
+};
+
+/**
+ * @param textInput user input word
+ * @returns html element <mark>textInput</mark>
+ */
+const highlightedWord = function markGivenWordToHighlight(textInput) {
+  return `<mark style="background-color: yellow">${textInput}</mark>`;
```

Comment:
This contains in-line CSS, which should be avoided. Just add a class instead and use the class to style.